functions/__sf_section_php.fish

```diff
@@ -0,0 +1,40 @@
+#
+# PHP
+#
+# PHP is a server-side scripting language designed primarily for web development.
+# Link: http://www.php.net/
+
+function __sf_section_php -d "Display the current php version"
+	# ------------------------------------------------------------------------------
+	# Configuration
+	# ------------------------------------------------------------------------------
+
+	__sf_util_set_default SPACEFISH_PHP_SHOW true
+	__sf_util_set_default SPACEFISH_PHP_PREFIX $SPACEFISH_PROMPT_DEFAULT_PREFIX
+	__sf_util_set_default SPACEFISH_PHP_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+	__sf_util_set_default SPACEFISH_PHP_SYMBOL "🐘 "
+	__sf_util_set_default SPACEFISH_PHP_COLOR blue
+
+	# ------------------------------------------------------------------------------
+	# Section
+	# ------------------------------------------------------------------------------
+
+	[ $SPACEFISH_PHP_SHOW = false ]; and return
+
+	if not test -f composer.json \
+		-o (count *.php) -gt 0
+		return
+	end
+
+	if not type -q php
```

Comment:
Wouldn't this be better to have before the test above? I don't have PHP installed, but it will always check for `composer.json` or a `*.php` file regardless. This seems inefficient performance-wise to me.

Reply:
Logically, it makes a bit more sense to check if PHP is installed _before_ checking the entire current directory for PHP-related files. Might be worth checking other sections to see if they [follow this guideline](https://github.com/matchai/spacefish/blob/master/CONTRIBUTING.md#will-it-slow-down-the-prompt), in the same way.

Reply:
Makes sense. I'll go ahead and fix that 👍